bin/receiver.py

```diff
@@ -59,7 +56,7 @@ def main():
         print('Cannot start SSM.  Pidfile %s already exists.' % pidfile)
         sys.exit(1)
 
-    ssm.agents.logging_helper(cp, options.log_config)
```

Comment:
Restore this line as we'll need to pass `log_config` on for now.

Reply:
does this line need to be restored? It looks like it wasn't but the conversation was resolved.

Reply:
Path check added to sender.py and receiver.py so we don't need to pass through the path in log_config.